core/modules/url_filter.cc

```diff
@@ -0,0 +1,279 @@
+#include "url_filter.h"
+
+#include <rte_ip.h>
+
+#include <algorithm>
+
+#include "../utils/ether.h"
+#include "../utils/http_parser.h"
+#include "../utils/ip.h"
+
+using bess::utils::EthHeader;
+using bess::utils::Ipv4Header;
+using bess::utils::TcpHeader;
+
+const uint64_t TIME_OUT = 12e10;
```

Comment:
`TIME_OUT_NS` might be better?
Also, if you intended 2 minutes, 120e9 would be a better representation